web3/providers/async_base.py

```diff
@@ -46,6 +50,7 @@ class AsyncBaseProvider:
     )
 
     is_async = True
+    has_persistent_connection = False
```

Comment:
I haven't thought too hard about this, but should this flag get put on the `BaseProvider` too? Or is it only here because of the inheritance chain?

Reply:
Good point. I think it does make sense to add it there for consistency.